docs/examples.rst

```diff
@@ -362,3 +362,77 @@ Calculate the token count from the token balance of an account address.
+    abi='[{"constant":true,"inputs":[],"name":"name","outputs":[{"name":"","type":"string"}],"payable":false,"stateMutability":"view","type":"function"},{"constant":false,"inputs":[{"name":"_spender","type":"address"},{"name":"_value","type":"uint256"}],"name":"approve","outputs":[{"name":"","type":"bool"}],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":true,"inputs":[],"name":"totalSupply","outputs":[{"name":"","type":"uint256"}],"payable":false,"stateMutability":"view","type":"function"},{"constant":false,"inputs":[{"name":"_from","type":"address"},{"name":"_to","type":"address"},{"name":"_value","type":"uint256"}],"name":"transferFrom","outputs":[{"name":"","type":"bool"}],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":true,"inputs":[],"name":"decimals","outputs":[{"name":"","type":"uint8"}],"payable":false,"stateMutability":"view","type":"function"},{"constant":true,"inputs":[{"name":"_owner","type":"address"}],"name":"balanceOf","outputs":[{"name":"","type":"uint256"}],"payable":false,"stateMutability":"view","type":"function"},{"constant":true,"inputs":[],"name":"symbol","outputs":[{"name":"","type":"string"}],"payable":false,"stateMutability":"view","type":"function"},{"constant":false,"inputs":[{"name":"_to","type":"address"},{"name":"_value","type":"uint256"}],"name":"transfer","outputs":[{"name":"","type":"bool"}],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":true,"inputs":[{"name":"_owner","type":"address"},{"name":"_spender","type":"address"}],"name":"allowance","outputs":[{"name":"","type":"uint256"}],"payable":false,"stateMutability":"view","type":"function"},{"anonymous":false,"inputs":[{"indexed":true,"name":"_from","type":"address"},{"indexed":true,"name":"_to","type":"address"},{"indexed":false,"name":"_value","type":"uint256"}],"name":"Transfer","type":"event"},{"anonymous":false,"inputs":[{"indexed":true,"name":"_owner","type":"address"},{"indexed":true,"name":"_spender","type":"address"},{"indexed":false,"name":"_value","type":"uint256"}],"name":"Approval","type":"event"}]'
+
+.. doctest::
+
+    >>> factory=web3.eth.contract(abi=json.loads(abi), bytecode=bytecode)
+    >>> tx_hash = factory.constructor(1000000, 'DEM', 18, 'DE', 1).transact()
+    >>> print(tx_hash)
+    b'\xe9K\xcfz\xfc9*-+\x17|\xd4at.\xe32t\x13\xf4\\\xe8\x8b\x9e6?\x08\xa1\xb4\xe6[\xd3'
+    >>> txn_receipt = web3.eth.waitForTransactionReceipt(tx_hash)
+
+
+Instantiate a contract pointing at a specific address
+-----------------------------------------------------
+
+.. doctest::
+
+    >>> new_contract = web3.eth.contract(address='0x89205A3A3b2A69De6Dbf7f01ED13B2108B2c43e7', abi=json.loads(abi))
```

Comment:
@iamonuwa one more, can you update all of these variables from `web3 -> w3` to stick with our convention which helps mitigate confusion between 1) the main module name and 2) an instance of the `Web3` instance.

Reply:
Because of the infura, w3 returns empty accounts

Reply:
Right, but we shouldn't be using infura anymore now that things should be wired up to the `EthereumTesterProvider`